### ISSUE TITLE
Refactor other pixel based design tokens etc.

### ISSUE THREAD

**Issue by jkaltoft** [kirbydesign/designsystem#3583]
This issue may require a bit of research and some UX/design decisions, since it reaches into layout and aesthetic details such as rounded corners, drop shadows, icons and avatars.

May require rethinking how and where these design tokens are applied

Other useful units we should consider include `dvw`, `cqw`, `ch` and `lh` to name a few, but there are *many* other units. Use what's appropriate.

Some values should still be declared using pixels, e.g. breakpoints

- [ ] Investigate and decide which tokens should be refactored. See e.g. `libs/core/src/scss/base/_variables.scss`. Are there more places we should look into?

-----

This issue is based on the findings of research https://github.com/kirbydesign/designsystem/issues/3568

Read more:
- [Success Criterion 1.4.4 Resize Text](https://www.w3.org/TR/WCAG22/#resize-text)
- [Understanding SC 1.4.4](https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html)
- About lenght units: [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/length) and [W3C](https://www.w3.org/TR/css-values-4/#lengths)

**Comment by RasmusKjeldgaard** [comment]
Between designers and developers it has been decided that our strategy for text resizing is to leave these aesthetic elements as-is for now. This is similar to how we try to preserve spacing etc. to enhance usability when text is scaled, by allowing as much content on the screen as possible/viable, instead of re-creating the airy look when resized.
